torch/testing/_internal/common_methods_invocations.py

```diff
@@ -15069,9 +15069,14 @@ def error_inputs_mean(op_info, device, **kwargs):
                    ),),
     OpInfo('lerp',
            dtypes=floating_and_complex_types(),
-           dtypesIfCUDA=floating_and_complex_types_and(torch.half, torch.bfloat16),
-           dtypesIfROCM=floating_and_complex_types_and(torch.half, torch.bfloat16),
+           dtypesIfCUDA=floating_and_complex_types_and(torch.chalf, torch.half, torch.bfloat16),
+           dtypesIfROCM=floating_and_complex_types_and(torch.chalf, torch.half, torch.bfloat16),
            sample_inputs_func=sample_inputs_lerp,
+           skips=(
+               # RuntimeError: "lerp_cuda" not implemented for 'ComplexHalf'
```

Comment:
Huh, this is a bad error msg, I think.

Can we make it more descriptive? (because `lerp` supports `chalf` here, as we can see).

Reply:
Looks like you will have to update lerp_scalar_kernel as well.

Reply:
Hi, @kshitij12345!
I tried that. But how to use scalar values in a jiterator string? (I think jiterator currently supports common data type inputs).

I got the following error:
```cpp
default_program(1509): error: identifier "weight" is undefined

default_program(1509): error: type name is not allowed

default_program(1509): error: expected an expression

3 errors detected in the compilation of "default_program".
```

This was my last commit [2556e14](https://github.com/pytorch/pytorch/pull/75584/commits/2556e148cc9964ef84ec6bb9d7489d36a878ac4e) for the same.

Your viewpoints, please... Thanks!

Reply:
Hi, @kshitij12345!
Yes, will update it. Forgot about that.